docs/components/newsletter-forms.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no">
+  <link rel="stylesheet" href="/cocoandeve-styleguides/css/main.critical.css">
+  <link rel="stylesheet" href="/cocoandeve-styleguides/css/main.css">
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+  <!-- <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/2.6.0/cjs/popper.min.js"></script> -->
+  <script src="/cocoandeve-styleguides/js/bootstrap.js"></script>
+  <script src="/cocoandeve-styleguides/js/main.js"></script>
+</head>
+
+<body class="mobile-wrapper">
+  <div class="container mt-4">
+    <h1>Newsletter Forms</h1>
+    <div class="newsletter-form">
+      
```

Comment:
it's empty. Also this must be a compounds since it uses a form component and a title component